lib/rules/rule-selector-property-disallowed-list/index.js

```diff
@@ -0,0 +1,69 @@
+// @ts-nocheck
```

Comment:
[suggestion] Can you remove `// @ts-nocheck` and pass the type-check, please?

Please refer to the following code:

https://github.com/stylelint/stylelint/blob/c0b1773f706614ed6acda8fa35264f918b81188c/lib/rules/alpha-value-notation/index.js#L24-L25

See also #4496